src/FSM/src/Runner/SequenceRunner.php

```diff
@@ -40,8 +40,7 @@ public function __construct(State $initialState, Stateful $context = null)
 
         $context->setCurrentState($initialState);
 
-        $this->originalContext = clone $this->context = $context;
```

Comment:
Refactored this as PHP 8 complained about the 2nd `=` can be used in write context only.